aspnetcore/blazor/class-libraries.md

```diff
@@ -39,6 +39,17 @@ Follow the guidance in the <xref:blazor/get-started> article to configure your e
    1. Right-click the app project. Select **Add** > **Reference**.
    1. Select the RCL project. Select **OK**.
 
+> [!NOTE]
+> If the **Support pages and views** check box is selected when generating the RCL from the template, the generated project doesn't include the *_Imports.razor* file with the following contents:
+>
+> ```razor
+> @using Microsoft.AspNetCore.Components.Web
+> ```
+>
+> Manually add the file the root of the generated project.
+>
+> This issue will be addressed in a future release of ASP.NET Core. For more information, see [RCL template either supports MVC OR supports Components (dotnet/aspnetcore #17870)](https://github.com/dotnet/aspnetcore/issues/17870).
```

Comment:
I don't think we need to link to the issue here in the docs. This isn't a blocking bug. Let's remove this line.